laser-py/laser.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# LASER  Language-Agnostic SEntence Representations
+# is a toolkit to calculate multilingual sentence embeddings
+# and to use them for document classification, bitext filtering
+# and mining
+#
+# --------------------------------------------------------
+#
+# Helper functions for tokenization and BPE
+
+import os
+import sys
+import logging
+import sentencepiece as spm
+from sacremoses import MosesPunctNormalizer, MosesDetokenizer
+
+logging.basicConfig(
+    stream=sys.stdout,
+    level=logging.INFO,
+    format="%(asctime)s | %(levelname)s | %(name)s | %(message)s")
+logger = logging.getLogger("preprocess")
+
+# get environment
+assert os.environ.get('LASER'), 'Please set the enviornment variable LASER'
+
+class LaserTokenizer:          
+    def tokenize(inp_fname, out_fname, spm_model_file, lang='en',
```

Comment:
Please provide type hints for each argument. For any file pointer lets use `pathlib.Path`.